deployments/values/supported-matrix.yaml

```diff
@@ -221,21 +221,21 @@ matrix:
     intelligence:
       version: "~1.5.0"
     trouter:
-      version: "~2.1.1"
+      version: "~4.1.1"
```

Comment:
not 100% sure if it's safe to reuse ga anchors but at least I would define new anchors instead of repeating versions here